test/integration/test_client_easy.c

```diff
@@ -0,0 +1,116 @@
+#include "../../include/dqlite.h"
+#include "../lib/munit.h"
+#include "../lib/runner.h"
+
+#include <dirent.h>
+#include <unistd.h>
+
+SUITE(client_easy);
+
+#define N_SERVERS 3
+
+struct fixture
+{
+	char *dirs[N_SERVERS];
+	dqlite_server *servers[N_SERVERS];
+};
+
+static void *setup(const MunitParameter params[], void *user_data)
+{
+	(void)params;
+	(void)user_data;
+	struct fixture *f = munit_malloc(sizeof *f);
+	unsigned i;
+	char dirpath[100];
+	char *p;
+	int rv;
+
+	for (i = 0; i < N_SERVERS; i += 1) {
+		strncpy(dirpath, "/tmp/dqlitetestXXXXXX", 100);
+		p = mkdtemp(dirpath);
+		if (p == NULL) {
+			munit_error("mkdtemp");
+		}
+		f->dirs[i] = strdup(dirpath);
+		rv = dqlite_server_create(dirpath, &f->servers[i]);
+		munit_assert_int(rv, ==, 0);
+	}
+
+	return f;
+}
+
+static void teardown(void *data)
+{
+	struct fixture *f = data;
+	unsigned i;
+	DIR *d;
+	struct dirent *de;
+
+	for (i = 0; i < N_SERVERS; i += 1) {
+		dqlite_server_destroy(f->servers[i]);
+		d = opendir(f->dirs[i]);
```

Comment:
Similarly, this could be:

```C
test_dir_tear_down(f->dirs[i]);
```

it will remove the directory recursively and free the allocated path string.